Makefile

```diff
@@ -60,7 +60,7 @@ TAGS        := $(if $(strip $(TAGS_JOINED)),-tags $(TAGS_JOINED),)
 
 ## Runs the test suite with mocks enabled.
 .PHONY: test
-test: test-bee2 tminio
+test: test-bee2 tminio unit-tests
```

Comment:
maybe run unit-tests first and then everything else

Reply:
Changed.